src/lib/messages.ts

```diff
@@ -75,12 +75,14 @@ export const messages = {
     },
     error: {
         analysisFailedGenerator: (reason: string) => `Analysis failed: ${reason}`,
+        engineUninstantiable: (engine: string) => `Heads up! We could not initialize the engine: "${engine}". Click "Learn More" for instructions on how to set up this engine to analyze your code.`,
```

Comment:
@jshackell-sfdc would love a review for tone, and to ensure this message is fully accurate @stephen-carter-at-sf

Reply:
```suggestion
        engineUninstantiable: (engine: string) => `Warning: We couldn't initialize the engine: "${engine}". Click "Learn More" for system requirements for this engine, and general instructions on how to set up Code Analyzer.`,
```

Reply:
Okay?  We don't have instructions for setting up specific engines, because it's pretty much the same for all engines.  The only difference is the prereq: i.e. Flowtest needs Python, PMD needs Java, and so on.

Reply:
That's much clearer - thank you!